scripts/__dropins__/tools/types/fetch-graphql/src/index.d.ts

```diff
@@ -92,6 +130,8 @@ declare class FetchGraphQLMesh {
             endpoint: string | undefined;
             fetchGraphQlHeaders: Header;
         };
+        addBeforeHook: (hook: BeforeHook) => void;
+        addAfterHook: (hook: AfterHook<any>) => void;
```

Comment:
Does your SDK build add more than just the configs and other lib files? This doesn't seem right.

Reply:
Sorry - what I meant to say was that since you were just changing locations of things, I did not expect source updates such as from fetch-graphql.

Reply:
I put that comment below though... 

> Should the SDK alpha build really include other changes not related to this? I think that means there is some build we have not yet integrated or tested, and so we should be testing those use cases too.